src/main/java/com/pnu/dev/radioserviceapi/mongo/Video.java

```diff
@@ -0,0 +1,26 @@
+package com.pnu.dev.radioserviceapi.mongo;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class Video {
```

Comment:
`RecommendedVideo` - the purpose of class should be explicit, we store only recommended videos.